schema/addresses/defs.yaml

```diff
@@ -0,0 +1,32 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+description: Common schema definitions for addresses schema
+"$defs":
+  typeDefinitions:
+    postcodetype:
+      description:
+        Granularity of the postcode from a finite, hierarchical, ordered
+        list of granularities. The lower the number of the postcode subtype,
+        the more precise it is in describing the locations encapsulated
+        within it.
+        
+        Not every country has to have each granularity of the postcode. 
+        Some countries have only one layer of postcodes, and some don't even use postcodes, like Angola, Bahamas, Botswana etc.
+      type: string
+      enum:
+        - postcode1       # Most granular, most detailed set of postcodes in a country.
+                          # E.g. in the UK: KT 10 9RT
+                          # And in Germany: 30657
+
```

Comment:
Are these phrases, "first-order" "second-order" postcodes commonly known? Could be - I'm just not familiar with that phrase.